packages/sdk-node/src/authenticateRequest.ts

```diff
@@ -1,16 +1,11 @@
 import type { RequestState } from '@clerk/backend';
-import { constants } from '@clerk/backend';
-import cookie from 'cookie';
```

Comment:
❓ Do we need the `cookie` package somewhere else in the `sdk-node` if not let's drop the dependency :)